lib/keystores.js

```diff
@@ -119,37 +129,58 @@ api.insert = async ({config} = {}) => {
  *   retrieve.
  * @param {Object} [options.query={}] - The query to use.
  * @param {Object} [options.options={}] - The query options (eg: 'sort').
+ * @param {boolean} [options.explain=false] - An optional explain boolean.
  *
- * @return {Promise<Array>} resolves to the records that matched the query.
+ * @return {Promise<Array> | ExplainObject} resolves to the records that matched
+ *   the query or an ExplainObject if `explain=true`.
  */
 api.find = async ({
-  controller, query = {}, options = {}
+  controller, query = {}, options = {}, explain = false
 } = {}) => {
   assert.string(controller, 'options.controller');
   // force controller ID
+  const collection = database.collections['kms-keystore'];
   query['config.controller'] = controller;
```

Comment:
I'd swap these two lines since the `// force controller ID` comment applies to the second one only, not setting this var.